.circleci/config.yml

```diff
@@ -64,3 +61,30 @@ jobs:
         path: /var/log/upstart/docker.log
     - store_artifacts:
         path: /tmp/circleci-test-results
+
+  build_java11:
+    docker:
+      - image: spotify/helios-test-container:1
+    working_directory: ~/spotify/helios
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      MAVEN_OPTS: -Xmx128m
+
+    steps:
+    - checkout
+
+    - run: ./circle.sh pre_machine
+
+    - run:
+        command: ./circle.sh dependencies
+        no_output_timeout: 20m
```

Comment:
I had to increase the timeout. A docker executor seems to have less resources and therefore is slower. Without this increased timeout, downloading deps often timed out and failed the build.

Reply:
could you add this as a comment into the config for future readers?

Reply:
https://github.com/spotify/helios/pull/1248